tests/rest_api/README.md

```diff
@@ -81,7 +81,7 @@ python utils/dump_objects.py
 To restore DB and data volume, please use commands below.
 
 ```console
-cat assets/cvat_db/cvat_db.dump | docker exec -i cvat_db pg_restore -1 -c -U root -d cvat
+cat assets/cvat_db/cvat_db.sql | docker exec -i cvat_db psql -U root -d cvat
```

Comment:
Should we drop DB before restoring it? Previously it was done using `-c` option. Also I believe in one transaction option was good (`-1` option).

Reply:
1. In README I mentioned that `cvat_db.sql` should be created by command:
```
docker exec cvat_db pg_dump -c -Fp -U root -d cvat > assets/cvat_db.sql
```
this command contains `-c` option and as result SQL instruction contains [drop objects part](https://github.com/openvinotoolkit/cvat/blob/sk/update-rest-api-tests/tests/rest_api/assets/cvat_db/cvat_db.sql#L19-L397)

2. `pg_restore` cannot restore db from `*.sql` file, but `psql` has not option `-1`